src/Avalonia.Controls/AutoCompleteBox.cs

```diff
@@ -252,6 +252,10 @@ public enum AutoCompleteFilterMode
     /// drop-down that contains possible matches based on the input in the text
     /// box.
     /// </summary>
+    [TemplatePart(ElementPopup,            typeof(Popup))]
+    [TemplatePart(ElementSelector,         typeof(SelectingItemsControl))]
+    [TemplatePart(ElementSelectionAdapter, typeof(ISelectionAdapter))]
```

Comment:
This stands out and seems to not be used in our templates.
I wonder if anybody has it in their custom templates.

Reply:
Not a problem for this PR anyway

Reply:
I'll take a look at this in more detail later. I notice quite a few things that should probably be cleaned-up/changed but all are breaking changes. What are you thoughts for breaking templates? Will it be allowed for 11.0?

Reply:
Yes, it's fine for 11.0. Just add label (or I will add) and left comments about breaking changes in description.
So later it can be copied to the change log.